package.json

```diff
@@ -34,6 +34,7 @@
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-select": "^5.7.3",
```

Comment:
Why are we adding react-select here? We haven't changed any frontend code?

Reply:
I assume that we accidentally have removed this dependency as part of [this PR](https://github.com/silverstripe/silverstripe-tagfield/pull/229), but we use `Select` component here
https://github.com/silverstripe/silverstripe-tagfield/blob/3/client/src/components/TagField.js#L2